### ISSUE TITLE
#DocReview: GenPipes Evolution history content needs review

### ISSUE THREAD

**Issue by shaloo** [c3g/GenPipes_Docs#10]
Review (&fact check) request:
Ref: GenPipes RTD documentation -> About -> GenPipes Evolution Infographic https://genpipes.readthedocs.io/en/latest/about/evolution.html

1. The infographic on this page uses data from published GenPipes paper.  However, the data for years 2018, 2019 in the chart is a placeholder.  That needs to be reviewed/replaced with actuals.

2. The infographic has been updated to incorporate Rola's initial inputs - ensure the updated infographic reflects her inputs.

**Comment by pphector** [comment]
I have the raw data for this infographic and I am in the process of cleaning it up so we have the final numbers for the infographic. I will update this shortly.

**Comment by pphector** [comment]
So, I went over the data now for 2018 and what we have so far for 2019, and here is the information for the infographic: 
 
2018:
- Servers: 14 (new mp2b)
- Pipelines: 15 (now counting protocols as separate pipelines)
- Runs: 9988

2019: 
- Servers: 15 (new beluga)
- Pipelines: 16
- Runs (so far): 4665 --> at this rate, by the end of the year, there will have been approx 8500, but it is very possible that this rate will increase towards the end of the year, so this number is not very predictive for how many runs there will be in the end. 

I hope this information helps!

**Comment by shaloo** [comment]
Thanks.  I have updated the information in the chart accordingly. Data for 2019 is shown in a dotted circle with an asterisk indicating that its partial data (June 2019).  See updates here https://genpipes.readthedocs.io/en/latest/about/history.html

best,
Shaloo

On 15-Jul-2019, at 4:34 AM, J. Hector Galvez <notifications@github.com<mailto:notifications@github.com>> wrote:


So, I went over the data now for 2018 and what we have so far for 2019, and here is the information for the infographic:

2018:

  *   Servers: 14 (new mp2b)
  *   Pipelines: 15 (now counting protocols as separate pipelines)
  *   Runs: 9988

2019:

  *   Servers: 15 (new beluga)
  *   Pipelines: 16
  *   Runs (so far): 4665 --> at this rate, by the end of the year, there will have been approx 8500, but it is very possible that this rate will increase towards the end of the year, so this number is not very predictive for how many runs there will be in the end.

I hope this information helps!
